haiku-apps/copynametoclipboard/copynametoclipboard-1.0.1.recipe

```diff
@@ -9,7 +9,10 @@ COPYRIGHT="2000 Werner Freytag
 	2012 Humdinger"
 LICENSE="MIT"
 REVISION="2"
-SOURCE_URI="$(HOMEPAGE)/archive/1da1d52429db53f15c3debf817e982d9ce6346b4.tar.gz"
+srcGitRev="1da1d52429db53f15c3debf817e982d9ce6346b4"
+SOURCE_URI="$HOMEPAGE/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="1da1d52429db53f15c3debf817e982d9ce6346b4"
```

Comment:
Ok I'll try that command. For some recipes, hp will tell me the checksum_sha256 to use, but not for these two recipes for some reason :/